src/components/EntryForm.js

```diff
@@ -23,114 +23,193 @@ export default function EntryForm({ onCreateCard }) {
+        </Part>
+        <Part>
+          <label htmlFor="timeto">To</label>
+          <Input
+            id="timeto"
+            name="timeto"
+            type="time"
+            step={1800}
+            onChange={handleOnChange}
+            required
+          />
+        </Part>
+      </Fieldset>
+      <small>water</small>
+      <Fieldset>
+        <Part>
+          <label htmlFor="water">water</label>
+          <Input
+            id="water"
+            name="water"
+            type="text"
+            maxLength={25}
+            onChange={handleOnChange}
+            placeholder="water"
+            required
+          />
+        </Part>
+        <Part>
+          <label htmlFor="stretch">Stretch</label>
+          <Input
+            id="stretch"
+            name="stretch"
+            type="text"
+            maxLength={100}
+            onChange={handleOnChange}
+          />
+        </Part>
+        <Part>
+          <label htmlFor="watertemp">Water temperature</label>
+          <Input
+            id="watertemp"
+            name="watertemp"
+            type="number"
+            min={-5}
+            max={50}
+            onChange={handleOnChange}
+            placeholder="°C"
+          />
+        </Part>
+        <Part>
+          <label htmlFor="watercolor">Water Color</label>
+          <select
+            id="watercolor"
+            name="watercolor"
+            type="text"
+            onChange={handleOnChange}
+          >
+            <option value="cloudy">cloudy</option>
+            <option value="normal">normal</option>
+            <option value="clear">clear</option>
+          </select>
+        </Part>
+        <Part>
+          <label htmlFor="waterlevel">Water Level</label>
+          <select
+            id="waterlevel"
+            name="waterlevel"
+            type="text"
+            onChange={handleOnChange}
+          >
+            <option value="low">low</option>
+            <option value="normal">normal</option>
+            <option value="high">high</option>
+          </select>
+        </Part>
+      </Fieldset>
+      <small>weather</small>
+      <Fieldset>
+        <Part>
+          <label htmlFor="weather">Weather</label>
+          <Input
+            id="weather"
+            name="weather"
+            type="text"
+            maxLength={100}
+            onChange={handleOnChange}
+          />
+        </Part>
+        <Part>
+          <label htmlFor="temperature">Temperature</label>
+          <Input
+            id="temperature"
+            name="temperature"
+            type="number"
+            min={-50}
+            max={50}
+            maxLength={100}
+            onChange={handleOnChange}
+            placeholder="°C"
+          />
+        </Part>
+        <Part>
+          <label htmlFor="airpressure">Air pressure</label>
+          <Input
+            id="airpressure"
+            name="airpressure"
+            type="number"
+            min={850}
```

Comment:
I don't know how important it is, that the min/max is accessible (at the moment it isn't), because I don't know anything about airpressure.

Reply:
just to make sure that they can not add stupid unrealistic data 🙃